client/src/slices/onboarding/onboardingSlice.ts

```diff
@@ -34,6 +34,12 @@ const onboardingSlice = createSlice({
       state.isCompleted = false
     },
   },
+  extraReducers: (builder) => {
+    builder.addCase('demo/resetState', (state) => {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      state = initialState
```

Comment:
I don't think this will work. If you modify a property on an object, the object is changed, but if you reassign the passed in value isn't actually changed. 

I think we can do it like this:

```suggestion
      return initialState
```